### ISSUE TITLE
jitsu deploy doesn't work

### ISSUE THREAD

**Issue by wclr** [nodejitsu/jitsu#514]
Is it because of node v0.12.2?

```
info:    Welcome to Nodejitsu username
info:    jitsu v0.15.0, node v0.12.2
info:    It worked if it ends with Nodejitsu ok
info:    Executing command deploy
info:    Analyzing application dependencies in ./server/index.js
timers.js:156
    throw new TypeError('msecs must be a number');
          ^
TypeError: msecs must be a number
    at Object.exports.enroll (timers.js:156:11)
    at TLSSocket.Socket.setTimeout (net.js:337:12)
    at ClientRequest.<anonymous> (_http_client.js:565:10)
    at ClientRequest.g (events.js:199:16)
    at ClientRequest.emit (events.js:129:20)
    at tickOnSocket (_http_client.js:493:7)
    at _http_client.js:504:7
    at process._tickCallback (node.js:355:11)
```

**Comment by abigwonderful** [comment]
getting the exact same error here. Surprised that there hasn't been a response.

```
timers.js:165
throw new TypeError('msecs must be a number');
      ^
TypeError: msecs must be a number
at Object.exports.enroll (timers.js:165:11)
at TLSSocket.Socket.setTimeout (net.js:337:12)
at ClientRequest.<anonymous> (_http_client.js:565:10)
at ClientRequest.g (events.js:199:16)
at ClientRequest.emit (events.js:129:20)
at tickOnSocket (_http_client.js:493:7)
at _http_client.js:504:7
at process._tickCallback (node.js:355:11)`
```

**Comment by mjgs** [comment]
I fixed the issue by downgrading to `node@0.10.x`

**Comment by indexzero** [comment]
Yes. `jitsu` does not work with `0.12` or `4.x` and we are not planning to upgrade it to do so. See:
- http://blog.nodejitsu.com/all-nodejitsu-services-extended-to-december/
- http://blog.nodejitsu.com/nodejitsu-modulus-transition/
